### PR TITLE
STY: Fix doc table line height

### LINKED ISSUE #575: DOC: CSS is messed up on some entries
For me this https://mne.tools/mne-bids-pipeline/features/steps.html looks like

Screenshot from 2022-07-28 19-08-48

### PR DESCRIPTION
**Pull request by larsoner**
Closes #575

@hoechenberger you added this in #543 where I see:

> Reduce table row height in example docs

Do you want to try pushing a different fix that doesn't cause the problem in #575, or just live with this revert?

### CODE REVIEW

**Comment by hoechenberger** [comment]
@larsoner This change was added to alter the "Demonstrated features" tables from this:
Screen Shot 2022-07-29 at 20 51 58

to this:
Screen Shot 2022-07-29 at 20 52 25

So maybe we need to apply a more restrictive CSS selector…

File: docs/source/css/extra.css [hunk 1]
```diff
@@ -13,7 +13,6 @@ p {
 .md-typeset table td,
 .md-typeset table th {
   font-size: 0.7rem;
-  line-height: 0.5;
```
Comment (drammock) [inline docs/source/css/extra.css:16]:
```suggestion
  padding: .5em 1.25em !important;
```

**Comment by larsoner** [comment]
Okay I didn't see any way to be more selective given the HTML, so I looked into wrapping it a bit. I figured I could add something to `gen_examples.py` to wrap the table in a div like `<div class="demonstrated-features-table">` but I can't get `mkdocs` render the markdown table *inside* that div.

@hoechenberger feel free to give it a shot if you know how. I was just going to change the `padding: .9375em 1.25em` to something smaller...

File: docs/source/css/extra.css (hunk 1 above)
Comment (larsoner) [reply docs/source/css/extra.css:16]: Yes we could do this but it's not selective. But if @hoechenberger is okay with affecting all tables I'm happy to commit

**Comment by hoechenberger** [comment]
All looking good. Thanks @larsoner!

**Pull request merged**